toc.js

```diff
@@ -45,6 +45,14 @@ module.exports = {
             'middleware-doc/doc/appendix-de-kassensichv/appendix-de-kassensichv',
             'middleware-doc/doc/appendix-de-kassensichv/terminology/terminology',
             'middleware-doc/doc/appendix-de-kassensichv/cash-register-integration/cash-register-integration',
+			 {
+              type: 'category',
+              label: 'Cash register integration',
```

Comment:
Let's move the "overview" about cash register integration to this category as well :) 
(from line 47 to line 52)